packages/salesforcedx-utils-vscode/package.json

```diff
@@ -29,7 +29,7 @@
     "@types/proxyquire": "1.3.28",
     "@types/shelljs": "0.7.9",
     "@types/sinon": "^2.3.7",
-    "@types/vscode": "1.72.0",
+    "@types/vscode": "^1.61.2",
```

Comment:
So 1.61 is Sept 2021. 1.72 is 09/2022  I intended to have the year old version for types in the bundling PR but must have lost track of making that update.  The types only update shouldn't be a problem b/c if it builds it builds.  Note we looked in app insights and verified that over 99% of our users are on a recent release. 

TL;DR.  1.61.2 is support back one year and covers the use case necessary for the bundling.